File: app/src/com/ternaryop/tumblr/TumblrException.java
package com.ternaryop.tumblr;

public class TumblrException extends RuntimeException {

	/**
	 * 
	 */
	private static final long serialVersionUID = -3382759637444771469L;


	public TumblrException() {
		super();
	}

	public TumblrException(String detailMessage, Throwable throwable) {
		super(detailMessage, throwable);
	}

	public TumblrException(String detailMessage) {
		super(detailMessage);
	}

	public TumblrException(Throwable throwable) {
		super(throwable);
	}
}

File: app/src/main/java/com/ternaryop/tumblr/TumblrException.kt
package com.ternaryop.tumblr

class TumblrException : RuntimeException {

    constructor(detailMessage: String) : super(detailMessage)

    constructor(throwable: Throwable) : super(throwable)

    companion object {

        /**
         *
         */
        private const val serialVersionUID = -3382759637444771469L
    }
}
